index.js

```diff
@@ -0,0 +1,56 @@
+const { declare } = require('@babel/helper-plugin-utils');
+
+const defaultTargets = ['> 1%', 'ie >= 11', 'Firefox ESR', 'last 2 versions'];
```

Comment:
@joesnellpdx @timwright12 I took this from the theme scaffold, is this list accurate?